added more tricks

prachisharma02 opened the pull request:
added more tips and format the text and changes made in the background to make it more attractive

prachisharma02 commented:
made first ever pull request

atapas commented:
@prachisharma02 Which issue is it related to? Please provide the issue link here.

prachisharma02 commented:
There was no issues i just added some more tricks as a contribution

On Wed, Oct 5, 2022, 8:05 AM Tapas Adhikary ***@***.***>
wrote:

> @prachisharma02 <https://github.com/prachisharma02> Which issue is it
> related to? Please provide the issue link here.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/atapas/html-tips-tricks/pull/11#issuecomment-1267837013>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2UNOQ53MCXAMCVXNOMMI7TWBTSP7ANCNFSM6AAAAAAQ4OEUU4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

prachisharma02 commented:
added more tricks please check it

atapas commented:
@prachisharma02 

Could you please list down the changes you have made?

If you have an issue for it, please update that and get me the link.

prachisharma02 commented:
https://github.com/atapas/html-tips-tricks/issues/39#issue-1403136640 -issue link
added two new tips 
* figure
* tooltip
* also changed the background and heading font

prachisharma02 commented:
please review it

prachisharma02 commented:
please provide a feedback
